src/components/enterprise-user-subsidy/data/hooks.jsx

```diff
@@ -68,7 +68,7 @@ export function useOffers(enterpriseId) {
   useEffect(
     () => {
       if (features.ENROLL_WITH_CODES) {
-        fetchOffers('full_discount_only=True', dispatch);
+        fetchOffers(`full_discount_only=True&enterprise_uuid=${enterpriseId}`, dispatch);
```

Comment:
Think it makes sense to add a unit test just covering this? I have had success with the `react-hooks-testing-library` which is used in some tests, e.g., https://github.com/edx/frontend-app-learner-portal-enterprise/blob/a979f4ad8fa63303a67b38378d6fbac9d0411cc0/src/components/course/data/tests/hooks.test.jsx#L27). Not clear how much work it will be for this one, let's explore if quick?